File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// AdviceSlip SDK exists test

require_once __DIR__ . '/../adviceslip_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = AdviceSlipSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: rb/test/exists_test.rb
# AdviceSlip SDK exists test

require "minitest/autorun"
require_relative "../AdviceSlip_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = AdviceSlipSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
